app/controllers/concerns/metadata.rb

```diff
@@ -62,6 +62,12 @@ def last_offset(total)
 
     private
 
+    def aggregated_count(count)
+      return count if count.is_a?(Integer)
+
+      count.values.sum
```

Comment:
Time to upgrade to Rails 6 and use [`reselect`](https://apidock.com/rails/ActiveRecord/QueryMethods/reselect)? :smile:

Reply:
It says:

> This is short-hand for unscope(:select).select(fields). Note that we’re unscoping the entire select statement.

So, @vkrizan, what exactly was your suggestion?

Reply:
Oh, my bad I haven't even noticed the unscope.

The suggestion was to use a db count. Even with the `unscope` it does not give the expected results:
```
=> Profile.with_score.unscope(:select).count
{"bdb7b0a0-439d-4fb2-894d-ff31479fe78f"=>1,
 "30080442-3bd4-4950-b744-4313b2a0620f"=>1,
...
```